### PR TITLE
Creating mobile menu

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
In this branch i created the mobile menu

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @Feliperto , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the required sections from the microverse readme template. Kindly update your readme file to have all the required sections (perequisite, install, usage and deployment).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/style.css [hunk 1]
```diff
@@ -67,13 +63,45 @@ body.scroll .navbar__section {
...
+.fa-times {
+  color: #fff;
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 62px 38px;
+  font-size: 16px;
+  z-index: 11;
+}
+
```
Comment (Strangeal) [inline css/style.css]: - [ ] Great job on implementing the hamburger menu as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address and fix.
  - [ ] To create a good user experience kindly consider increasing the size of the close button (X) to match that of the `Figma` design as it's currently too small. 
  - [ ] Also, Consider reducing the empty space you have above the nav text as it's too much and doesn't match that of the `Figma` design. See below for more details:
  
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-06-20 at 15 09 17
</td>
<td>
Screenshot 2023-06-20 at 15 07 24
</td>
</tr>
</table>

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Feliperto, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @Feliperto  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**